apis/v1alpha2/gateway_types.go

```diff
@@ -334,9 +334,13 @@ const (
 	// In this mode, TLS session between the downstream client
 	// and the Gateway is terminated at the Gateway.
 	TLSModeTerminate TLSModeType = "Terminate"
+
 	// In this mode, the TLS session is NOT terminated by the Gateway. This
 	// implies that the Gateway can't decipher the TLS stream except for
 	// the ClientHello message of the TLS protocol.
+	//
+	// Note that SSL passthrough is not supported for HTTPRoute. If you're
+	// looking to implement that kind of functionality use TLSRoute.
```

Comment:
Similar to comment on FAQ below, maybe this can be simplified to only mention TLSRoute:
```suggestion
	// Note that SSL passthrough is only supported by TLSRoute.
```